### ISSUE TITLE
[WP6.1] Calender and Archive blocks are broken when fontFamily is changed

### ISSUE THREAD

**Issue by t-hamano** [WordPress/gutenberg#44530]
### Description

Originally reported in trac: https://core.trac.wordpress.org/ticket/56678

The following two blocks, which support typography, cause errors when changing the font family.

- core/archives
- core/calendar

This does not occur when the gutenberg plugin is enabled, nor in other blocks that support font families. I expect this is a problem unique to dynamic blocks using `ServerSideRender`.

### Step-by-step reproduction instructions

- Insert the calender block or the archives block.
- Change the font-family from Typograph panel.
- Confirm that the block is broken.

### Screenshots, screen recording, code snippet

https://user-images.githubusercontent.com/54422211/192792592-ca1e6b62-8c18-463e-ac7b-a696fa63244b.mp4

The browser console records a 500 error in the API request and React errors:

![console_error](https://user-images.githubusercontent.com/54422211/192792564-11d830e6-bb97-413d-8413-40dd3548f313.png)

### Environment info

- WordPress version: 6.1-beta2-54344 (without Gutenberg plugin)

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by t-hamano** [comment]
It is probably a REST API issue, as the endpoint displays the following error:

```
{
	"code": "rest_invalid_param",
	"message": "Invalid parameter(s): attributes",
	"data": {
		"status": 400,
		"params": {
			"attributes": "fontFamily is not a valid property of Object."
		},
		"details": {
			"attributes": {
				"code": "rest_additional_properties_forbidden",
				"message": "fontFamily is not a valid property of Object.",
				"data": null
			}
		}
	}
}
```

**Comment by t-hamano** [comment]
This issue has been resolved in #43937 and is closed.
I will also comment in #43937 whether this PR should be backported to WordPress 6.1.
